cmd/restic/cmd_prune.go

```diff
@@ -404,9 +432,22 @@ func Prune(opts PruneOptions, gopts GlobalOptions, repo restic.Repository, usedB
 	Verbosef("unused size after prune: %s of total size\n\n",
 		formatPercent(stats.size.unused-stats.size.remove-stats.size.repackrm, totalSize-totalPruneSize))
 
-	Verbosef("total data files: %d / keep used: %d, keep partly used: %d, repack: %d, delete: %d, delete unreferenced: %d\n\n",
-		int(stats.packs.keepUsed+stats.packs.keepPartlyUsed)+len(repackPacks)+len(removePacks)+len(removePacksFirst),
-		stats.packs.keepUsed, stats.packs.keepPartlyUsed, len(repackPacks), len(removePacks), len(removePacksFirst))
+	Verbosef("mixed data files:        %10d\n", stats.packs.mixed)
+	Verbosef("tree data files:         %10d\n", stats.packs.tree)
+	Verbosef("data data files:         %10d\n", stats.packs.data)
+	Verbosef("unreferenced data files: %10d\n\n", len(removePacksFirst))
+	Verbosef("total data files:        %10d\n\n",
+		stats.packs.mixed+stats.packs.tree+stats.packs.data+uint(len(removePacksFirst)))
+
+	Verbosef("small data files:       %10d\n", stats.packs.small)
+	Verbosef("totally used data files: %10d\n", stats.packs.used)
+	Verbosef("partly used data files: %10d\n", stats.packs.partlyUsed)
+	Verbosef("unused data files:      %10d\n", stats.packs.unused)
+
+	Verbosef("to keep:   %10d data files", stats.packs.keep)
+	Verbosef("to repack: %10d data files", len(repackPacks))
+	Verbosef("to delete: %10d data files", len(removePacks))
+	Verbosef("to delete: %10d unreferenced data files", len(removePacksFirst))
```

Comment:
I think you missed \n here..

Reply:
True - I reformated the output and didn't test after. Shame on me... :wink: